src/playground/sample_data_tooling/sample_data_generator/sample_data_generator.py

```diff
@@ -50,23 +53,28 @@ def choose_field(kind, args = None):
     # To see more about Community Providers, visit:
     # https://faker.readthedocs.io/en/master/communityproviders.html
     # and follow instructions there to install them on your environment
-    # 
+    #
     # Example code snippet using the Biology Provider:
-    # 
+    #
     # from faker_biology.physiology import CellType, Organ, Organelle
     # fake.add_provider(CellType)
     # fake.add_provider(Organ)
     # fake.add_provider(Organelle)
+    #
+    # Another example code snipped using the School Provider:
```

Comment:
typo: snippet

Reply:
line 64 to 67 should be removed right?

Reply:
Yes, they should and I have removed them.